### PR TITLE
Dev/ste flor/handle metab prefixsuffix

### PR DESCRIPTION
**Pull request by SteFlor**

### CODE REVIEW

**Comment by Unaimend** [comment]
@SteFlor Could you provide three or four tests to see how it works :) Just a few examples to see what the RegExs do :)

**Comment by SteFlor** [comment]
There is a little bit of testing already! But I can expand it if needed!

Il giorno gio 16 nov 2023 alle 20:29 Thomas Dost ***@***.***>
ha scritto:

File: src/handle_metabolites_prefix_suffix.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            # go to the index of 'cpd' + 3(the length) of cpd because we now expect
+            # the numerical id
+            for i in range(cpd_index + 3, len(met_id)):
+                # only append the digits and stop as soon as you encounter a non-digit
+                if met_id[i].isdigit():
+                    identifier += met_id[i]
+                else:
+                    break
+    else: # remove 'M_' prefix and compartment string from metabolite id, preserving content between them.
+        identifier = re.sub("^M_(.*)_[a-z]\d?$","\g<1>", met_id)
```
Comment (Unaimend) [inline src/handle_metabolites_prefix_suffix.py:33]: Please describer in words hat those do

**Comment by Unaimend** [comment]
I indeed did not scroll far enough xD Except the comment I made it looks good from my side

**Review by Unaimend — APPROVED** [review]
Looks good :+1:

**Comment by Unaimend** [comment]
:1st_place_medal:

**Pull request merged**